charts/civil-sdt/values.preview.template.yaml

```diff
@@ -15,7 +15,14 @@ java:
     civil:
```

Comment:
Key vault name should be civil-sdt.  The main branch has the correct name, so I think this branch needs to be updated with the changes from main.